.github/workflows/triage.yml

```
@@ -45,8 +45,13 @@ jobs:
           )"
           echo "workflow_modified=${workflow_modified}" >> "${GITHUB_OUTPUT}"
 
+      # Wait briefly in case of failure to make sure we don't end up
+      # hitting the same API error when trying `gh pr edit`.
+      - if: failure()
+        run: sleep 30
+
       - name: Label PR
-        if: fromJson(steps.files.outputs.workflow_modified)
+        if: fromJson(steps.files.outputs.workflow_modified) || !success()
```

Not sure if it will work correctly if `workflow_modified` is undefined due to API error. Maybe this?

```suggestion
        if: !success() || fromJson(steps.files.outputs.workflow_modified)
```

---

Yea, that's better. Needs to be wrapped in `${{ }}` or `" "`, though, since `!` is a special character at the beginning of YAML entries.